### ISSUE TITLE
Add a start reversed option on large glass panels to make them grating panels.

### ISSUE THREAD

**Issue by ZombiePwner1** [BEEmod/BEE2-items#2023]

**Comment by Luke18033** [comment]
Interesting idea.

**Comment by Endermage77** [comment]
Haven't I already suggested this one before?
It was #1590, I believe.

**Comment by HelloIamapersonthatisalive** [comment]
I like

**Comment by ZombiePwner1** [comment]
@Endermage77 Yes you have suggested it but that's for regular angled panels this issue is for the large glass panels to form bigger grating panels.

**Comment by Endermage77** [comment]
Oh. What would a large grating panel even look like, anyway?
The regular-sized grating borders can be pretty much stretched to any size and still be regular grating, so I am curious how this would differ from the standard.

**Comment by ZombiePwner1** [comment]
@Endermage77 A few large grating panels make an appearance in the coop campaign mainly course 2 mass and velocity.
